includes/class-simple-local-avatars.php

```diff
@@ -343,7 +357,7 @@ public function get_simple_local_avatar_url( $id_or_email, $size ) {
 		}
 
 		// Fetch local avatar from meta and make sure it's properly set.
-		$local_avatars = get_user_meta( $user_id, $this->user_key, true );
+		$local_avatars = self::get_user_local_avatar( $user_id );
```

Comment:
If we change the above method to not be static, this will need updated:
```suggestion
		$local_avatars = $this->get_user_local_avatar( $user_id );
```